src/components/dashboard/StockItem.js

```diff
@@ -28,44 +25,42 @@ const useStyles = makeStyles({
   },
   pos: {
     marginBottom: 12,
-  },
-  delete: {
-    marginLeft: "auto"
   }
 });
 
 export default function StockItem({stock, delStock}) {
   const classes = useStyles();
 
+  function getTitle(title) {
+    if(title) {
+      if(title.length <= 23) {
```

Comment:
Is there a particular reason why 23 characters is the limit? Does anything over 23 characters cause styling/overflow issues?

I'm probably splitting hairs here, but font size and type (e.g. monospaced vs. not monospaced) could also affect the width of the rendered string, so string length may not be entirely representative of the "badness" of the given string.

Buuuut, the company titles are all ASCII, and we _probably_ won't see any super long titles, so this is more of a curiosity thing than an actual issue.

Reply:
I just kinda just played around with the code a little until I saw what looked the best especially since I believe the width of the cards remain the same. I didn't want to overcomplicate things.

Reply:
Okay, sounds reasonable enough to me. LGTM, I'm gonna go ahead and merge.